src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 30px 14px 14px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 45px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card__code {
+  width: 100%;
+  text-align: left;
+  margin-bottom: 15px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+}
+
+.card__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__stars--4 .card__star:nth-child(-n+4) {
```

Comment:
add classes for all cases(-n+1, -n+2 etc). this cars should be an example of working scenario, not hardcoded only four stars version